products/facilities/regional-offices/engineering/testing-qa.md

```diff
@@ -1 +1,90 @@
+
+Generally, the simplicity of the page comes from:
+1. Unauthenticated content
+2. Unidirectional flow of content
+3. Uses shared templates from other products as much as possible
+4. Uses design system components as much as possible
+
+The complexity of the page comes from:
+1. React Component with API Calls (mapbox and vets-api) for Nearby VA Locations and Dynamic Content (e.g. if vets-api returns no close locations, we show the user a message indicating that fact rather than a list of far locations.)
+2. Branching logic whether to display certain content on the page (e.g. if editor does not provide "Prepare for your visit" content we do not show the section)
+
+React Component complexity:
+The interactive complexity is simple, but the API dependence adds to its comprehensive complexity. Component testing with mock data is added for all scenarios.
+
+The interactive elements on the page are limited to **accordions**, **links** and occasionally **operating status notifications**:
+There are two forms of accordion sets on the page, one for "Prepare for your visit" (very limited complexity) and a few sets of "Service" accordions (moderate complexity). The "Prepare for your visit" accordion items represent a list of content that comes from WYSIWYG CMS editor content without any processing in content-build. The "Service" accordions are migrating to a centralized design with VAMC which now also have adopted the pattern of using the Service Location Paragraph type from CMS. While this makes them more complex than the WYSIWYG content from CMS, they contain logic that is tested (see below about accordion testing). As we migrate VAMC to the shared Service Location Paragraph types we are consolidating automated testing and ensuring correctness with manual UX testing.
+
+Links can be found throughout the page and, when possible, are using va-link components ensuring their reliability. However, in general, most links come from CMS content and make it difficult to test in an automated way, therefore it is the responsibility of the editor to ensure links are correct.
+
+Unchanging elements on the page are limited to **spotlight cards**, **Benefits Hotline alert**, **Updates links container**, **Office hours**, are components when possible (e.g. alerts), and generally are just containers for links. These elements are fairly simple, but do often depend on content from the CMS as well.
```

Comment:
Is "updates links container" a typo? I can't find the Figma to double check. :/

Reply:
I meant to really say the: "Get updates from the Veteran Benefits Administration" container, but don't know if it really has a name.

Reply:
https://www.figma.com/file/8RYmYfVGS9CUfZbomH5lBe/VBA-Regional-Office-Prototype?type=design&node-id=3-10371&mode=design&t=QDXczqhNZzj7hDZA-0